package/fedora/freecad.spec

```diff
@@ -197,13 +221,29 @@ LDFLAGS='-Wl,--as-needed -Wl,--no-undefined'; export LDFLAGS
        -DCMAKE_INSTALL_INCLUDEDIR=%{_includedir} \
        -DRESOURCEDIR=%{_datadir}/%{name} \
        -DFREECAD_USE_EXTERNAL_PIVY=TRUE \
+%if 0%{?fedora} > 38
+       -DFREECAD_USE_EXTERNAL_FMT=TRUE \
+%endif
```

Comment:
this defaults to on so it shouldn't be necessary, not a problem to explicitly set it though.

Reply:
I had some problems with the old grouping, so set it explicitly.

Reply:
Version check removed